src/util/tr.js

```diff
@@ -877,6 +943,66 @@ const _tr = (msg, {
     })
   }
 
+  const triggerByInactivity = (targetNotif) => {
+    const IDLE_TIME_THRESHOLD = targetNotif.display.deliveryTrigger.inactive * 1000 // 5 seconds
+    let idleTimer
+
+    function removeEventListeners () {
+      window.removeEventListener('mousemove', resetIdleTimer)
+      window.removeEventListener('keypress', resetIdleTimer)
+      window.removeEventListener('scroll', resetIdleTimer)
+      window.removeEventListener('mousedown', resetIdleTimer)
+      window.removeEventListener('touchmove', resetIdleTimer)
+      window.removeEventListener('click', resetIdleTimer)
+    }
+
+    window.addEventListener('mousemove', resetIdleTimer)
+    window.addEventListener('keypress', resetIdleTimer)
+    window.addEventListener('scroll', resetIdleTimer)
+    window.addEventListener('mousedown', resetIdleTimer)
+    window.addEventListener('touchmove', resetIdleTimer)
+    window.addEventListener('click', resetIdleTimer)
```

Comment:
Both addEventListener and removeEventListener can be optimised to something like 
const events = ['mousemove', 'keypress', 'scroll', 'mousedown', 'touchmove', 'click'];
  const addEventListeners = () => events.forEach(event => window.addEventListener(event, resetIdleTimer));
  const removeEventListeners = () => events.forEach(event => window.removeEventListener(event, resetIdleTimer));

Reply:
Done